Modules/_testcapi/mem.c

```diff
@@ -613,5 +613,14 @@ _PyTestCapi_Init_Mem(PyObject *mod)
         return -1;
     }
 
+#ifdef WITH_MIMALLOC
+    v = Py_True;
+#else
+    v = Py_False;
+#endif
+    if (PyModule_AddObjectRef(mod, "WITH_MIMALLOC", v) < 0) {
```

Comment:
Please log this value in Lib/test/pythoninfo.py, around WITH_PYMALLOC.

Note: pymem_getallocatorsname is already logged as pymem.allocator key, that's fine.